### PR TITLE
feat(ex_cmds)!: remove :behave

### PR DESCRIPTION
**Pull request by bfredl**
just use the individual options instead.

    set selection=exclusive
    set selectmode=mouse,key
    set mousemodel=popup
    set keymodel=startsel,stopsel

### CODE REVIEW

File: runtime/doc/usr_09.txt [hunk 1]
```diff
@@ -124,22 +124,17 @@ This adds the 'l' flag to 'guioptions'.
...
 select text in a standard manner.  The X Window system also has a standard
 system for using the mouse.  Unfortunately, these two standards are not the
-same.
-   Fortunately, you can customize Vim.  You can make the behavior of the mouse
-work like an X Window system mouse or a Microsoft Windows mouse.  The following
-command makes the mouse behave like an X Window mouse: >
+same. Fortunately, you can customize Vim.
 
-	:behave xterm
+The default behavior of the mouse on Unix systems is xterm.
```
Comment (zeertzjq) [inline runtime/doc/usr_09.txt:129]: This is wrong after #19290.

File: runtime/doc/usr_09.txt [hunk 2]
```diff
@@ -124,22 +124,17 @@ This adds the 'l' flag to 'guioptions'.
...
-details about what the two behaviors are, see |:behave|.  Here follows a
-summary.
+	set selection=exclusive
+	set selectmode=mouse,key
+	set mousemodel=popup
+	set keymodel=startsel,stopsel
 
+Here follows a summary.
 
 XTERM MOUSE BEHAVIOR
```
Comment (zeertzjq) [inline runtime/doc/usr_09.txt:140]: This summary looks out of place without `:behave`

**Comment by ErichDonGubler** [comment]
Is it worth mentioning that `behave xterm` can be migrated to with a mere `set mousemodel=extend`?

**Comment by dundargoc** [comment]
Can't hurt. Wanna send a PR?

**Comment by ErichDonGubler** [comment]
@dundargoc: [Sent!](https://github.com/neovim/neovim/pull/24024)

**Pull request merged**